netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -679,12 +685,14 @@ void returnProcessedBytes(Http2Stream http2Stream, int bytes) {
 
   private void closeStreamWhenDone(ChannelPromise promise, int streamId) throws Http2Exception {
```

Comment:
Hmm... All callers to this method have the TransportState, but it is hidden behind the StreamIdHolder interface. In the future we should probably just have the callers pass the object in, but that'd require some changes to the commands.

Reply:
No, we'd just expose TransportState directly from the commands. The commands are just for `Netty*Stream` and `Netty*Handler` to communicate.